### DISCUSSION TITLE
Getting Started

### DISCUSSION THREAD

**Discussion by KaiULeth** [minaoar/library-version-wise-opinion-extration#28] [General]
Hello. I am trying to get started with this project and have been attempting to follow the README file. I've ran into a couple issues which I am not sure if it may be Docker related or not. With the project directory cloned and running the command for step 3 I ran into an error for the legacy install. This is also causing errors with the Spacy import but has installed everything else besides Spacy. Do you have any advice on what I may be doing incorrectly with this?

**Comment by minaoar** [comment]
Hi, thanks a lot for your interest in the project :)

Regarding the issue you are facing, Have you installed docker?
In the step#3, did you include the . (period)?
It’s actually hard to diagnose with this much information.

**Reply by KaiULeth** [reply]
Sorry for the lack of information. I do have Docker installed, however, Docker is providing me with a fail to start error, which I'm not familiar with on how to resolve. I've redownloaded it and also enabled Windows virtualization, which was what I could determine needed to be done based on some Google research into the issue.
One of my colleagues is running Docker through a VM and running into the legacy install and Spacy import issue I described above. They have executed the command with both the period and without, though I believe without they ran into additional errors. 

Don't hesitate to ask some more questions regarding this. I'd like to provide you with the information you need to give me an opinion on this.

**Reply by minaoar** [reply]
In fact, Docker was not mandatory, it was just more helpful than installing directly in Windows.
You can also try directly installing the dependencies by running requirements.txt. If you see so many errors, then Docker is the only option I suppose. 
When you are trying Docker, I would say can you run a simple python application from online into your Docker first, just to be sure that Docker is installed and running properly? If we are sure that there is no issue in Docker, then we can investigate application specific issues. Thanks.

**Reply by KaiULeth** [reply]
Thanks for the help! I've got it running with Docker now

**Reply by minaoar** [reply]
Cool, glad to know 😎
